nservicebus/sagas/index.md

```diff
@@ -21,6 +21,18 @@ Design processes with more than one remote call to use sagas.
 
 While it may seem excessive at first, the business implications of the system getting out of sync with the other systems it interacts with can be substantial. It's not just about exceptions that end up in the logs.
 
+## No saga instance available
+
+If a message is received while there is no saga instance, then by default this message gets discarded and succesfully processed. When you have a saga instance that is already completed then it makes sense to discard other messages.
+
+This can also happen for saga instance creation. A message is received that expects a saga instance to be there but that message is not received yet. In this case the message should not be discarded. There are two options to resolve this:
```

Comment:
> This can also happen for saga instance creation.

I think this sentence isn't understandable by itself. Start with something like "This behavior" instead of just "This", but without the next sentence, it's not clear how discarding messages can happen for messages that start a saga?

Reply:
> A message is received that expects a saga instance to be there but that message is not received yet

two times referring `a message`, it's not super clear that the second mentioned message is a different message.

Reply:
> In this case the message should not be discarded.

but this is the current behavior. Maybe restate the behavior to something like "There are two options to avoid discarding incoming messages in this scenario"?